multimodal/requirements_test.txt

```diff
@@ -1,10 +1,14 @@
 black==22.1.0
```

Comment:
Do we need this package?

Reply:
I don't think so.

Reply:
Fixed in [f5564e6](https://github.com/microsoft/hi-ml/pull/438/commits/f5564e6bccfed8fca128175108a9877bad9b7557).